documentai/snippets/handle_response_sample.py

```diff
@@ -162,17 +172,39 @@ def print_image_quality_scores(
         print(f"        {detected_defect.type_}: {detected_defect.confidence:.1%}")
 
 
-def print_styles(styles: Sequence[documentai.Document.Style], text: str) -> None:
-    print(f"    {len(styles)} styles detected:")
-    first_style_text = layout_to_text(styles[0].layout, text)
-    print(f"        First style text: {repr(first_style_text)}")
-    print(f"           Color: {styles[0].color}")
-    print(f"           Background Color: {styles[0].background_color}")
-    print(f"           Font Weight: {styles[0].font_weight}")
-    print(f"           Text Style: {styles[0].text_style}")
-    print(f"           Text Decoration: {styles[0].text_decoration}")
-    print(f"           Font Size: {styles[0].font_size.size}{styles[0].font_size.unit}")
-    print(f"           Font Family: {styles[0].font_family}")
+def print_style_info(style_info: documentai.Document.Page.Token.StyleInfo) -> None:
+    """
+    Only supported in version `pretrained-ocr-v2.0-2023-06-02`
+    """
+    print(f"           Font Size: {style_info.font_size}pt")
+    print(f"           Font Type: {style_info.font_type}")
+    print(f"           Bold: {style_info.bold}")
+    print(f"           Italic: {style_info.italic}")
+    print(f"           Underlined: {style_info.underlined}")
+    print(f"           Handwritten: {style_info.handwritten}")
+    print(
+        f"           Text Color (RGBa): {style_info.text_color.red}, {style_info.text_color.green}, {style_info.text_color.blue}, {style_info.text_color.alpha}"
+    )
+
+
+def print_visual_elements(
+    visual_elements: Sequence[documentai.Document.Page.VisualElement], text: str
+) -> None:
+    """
+    Only supported in version `pretrained-ocr-v2.0-2023-06-02`
```

Comment:
Will it continue to only be supported in this one specific version? Or should we generalize this to say something like "Only supported in version `pretrained-ocr-v2.0-2023-06-02` and later"?

Reply:
The product team hasn't confirmed one way or another. There's no guarantee it'll be supported in later versions.